libs/ui/src/lib/input/numbers-only.directive.ts

```diff
@@ -0,0 +1,17 @@
+import { Directive, ElementRef, HostListener, Input } from '@angular/core';
+
+@Directive({
+  // eslint-disable-next-line @angular-eslint/directive-selector
+  selector: '[numbersOnly]',
```

Comment:
Shouldn't we prefix any directive selector with `cvi`? Or, is it something for internal usage? Also, if it's not meant to be reused by different components, should it stay a directive or should it be incorporated to `InputComponent` as a prop?

Reply:
Added cvi prefix. It is meant for id (not personal code) input.